test/show_test.rb

```diff
@@ -0,0 +1,36 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/character'
+require './lib/show'
+require 'pry'
+
+class ShowTest < Minitest::Test
+
+  def test_show_exist
+    kitt = Character.new({name: "KITT", actor: "William Daniels", salary: 1_000_000})
+    michael_knight = Character.new({name: "Michael Knight", actor: "David Hasselhoff", salary: 1_600_000})
+    knight_rider = Show.new("Knight Rider", "Glen Larson", [michael_knight, kitt])
+
+    assert_instance_of Show, knight_rider
+  end
+
+  def test_attributes
+    kitt = Character.new({name: "KITT", actor: "William Daniels", salary: 1_000_000})
+    michael_knight = Character.new({name: "Michael Knight", actor: "David Hasselhoff", salary: 1_600_000})
+    knight_rider = Show.new("Knight Rider", "Glen Larson", [michael_knight, kitt])
```

Comment:
usually after the second time of doing the same setup, you'll want to move these lines up into a `def setup` method